src/assets/styles/20-tools/_mixins/_typography.scss

```diff
@@ -27,25 +27,55 @@
  * Generates a style where a horizontal line displays
  * beneath an element on hover.
  */
-@mixin animated-underline {
+@mixin animated-underline($use-deprecated: false) {
   --transition-duration: 0.3s;
-  position: relative;
+  --transition-timing-function-in: cubic-bezier(0.23, 1, 0.32, 1);
+  --transition-timing-function-out: cubic-bezier(0.55, 0.085, 0.68, 0.53);
 
-  &:before {
-    background-color: var(--colour-white);
-    bottom: 0;
-    content: '';
-    height: calc(0.125 * var(--space-unit));
-    left: 0;
-    position: absolute;
-    transform: scaleX(0);
-    transition: all var(--transition-duration) ease-in-out;
-    visibility: hidden;
-    width: 100%;
+  /**
+   * We are currently maintaining two very similar styles, and are in the
+   * middle of migrating between the two.
+   */
+  @if not index((true, false), $use-deprecated) {
+    @warn '$use-deprecated parameter must be a boolean';
   }
 
-  &:hover:before {
-    transform: scaleX(1);
-    visibility: visible;
+  @if $use-deprecated == true {
+    --transition-timing-function-in: ease-in-out;
+
+    position: relative;
+
+    &:before {
+      background-color: var(--colour-black);
+      bottom: 0;
+      content: '';
+      height: calc(0.125 * var(--space-unit));
+      left: 0;
+      position: absolute;
+      transform: scaleX(0);
+      transition: all var(--transition-duration) var(--transition-timing-function-in);
+      visibility: hidden;
+      width: 100%;
+    }
+
+    &:hover:before {
+      transform: scaleX(1);
+      visibility: visible;
+    }
+  }
```

Comment:
Want to check in with @j-jaworski on this one... we're introducing a new, v similar style with the animated underline (imitated from British Museum website) to one we currently use in the footer nav links.

Will check in on Monday to see if we want to replace the footer nav link style with the new updated one, or whether we need to maintain both. I'll then update code accordingly.

Reply:
We should change the footer hover state to animate from the left as per the new style but think it will look weird if they are exactly the same (thicker line used on card hovers might look a bit OTT on the footer's smaller text links)

Reply:
Ah amazing, wasn't expecting you to respond on here - was gonna flag on Slack. Thanks for that update, v helpful - will keep the line thickness the same as it currently is on footer links (think it's probs 1px).

Reply:
1px sounds right, feel free to make everything else the same, except colour obvs

Reply:
https://github.com/wellcometrust/corporate/issues/7729